### PR TITLE
Fix qemu section issues

### PR DESCRIPTION
**Pull request by kprosise**
Changes made for consistency, readability, and to address missing information such as version info.

QA Steps: checked for correct output.

This commit addresses FFTK-3196

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Combines issues mentioned in documentation Slack channel with those found while
in the process.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 8cf7b5f5eeef98d26c597234eacb8f2913e1970e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2660/docs/artifacts/html/index.html

File: source/user-guide/qemu/arm.rst [hunk 1]
```diff
@@ -9,6 +9,9 @@ arm
 QEMU CLI
 ^^^^^^^^
 
+.. important::
```
Comment (mike-scott) [inline source/user-guide/qemu/arm.rst:12]: There's another `qemu-system-arm` block above this one that also includes the `-no-acpi`.  That block begins with:
```
Use .qcow2 if you wish to change the QEMU disk size
```

Comment (mike-scott) [reply source/user-guide/qemu/arm.rst:12]: The block I mentioned above also shows:
```-drive id=disk0,file=|MACHINE|_device1.qcow2```
It looks like the `|MACHINE|` substitution isn't working there?

Comment (mike-scott) [reply source/user-guide/qemu/arm.rst:12]: Same for the arm64 and x86_64 pages as well.

File: source/user-guide/qemu/arm64.rst [hunk 2]
```diff
@@ -19,7 +22,9 @@ QEMU CLI
        -chardev null,id=virtcon -machine virt,secure=on -nographic
 
 .. note::
-    The |FIRMWARE_BLOB| artifact can typically be found in the same location where you downloaded the ``.wic.gz`` image. If you are unable to locate |FIRMWARE_BLOB|, consider checking for other artifacts such as; ``QEMU_EFI.fd`` and ``QEMU_VARS.fd``. These can be used to boot the image with the ``-drive`` flag.
+    A |FIRMWARE_BLOB| artifact is usally in the same location where you downloaded the ``.wic.gz`` image.
+    If you are unable to locate |FIRMWARE_BLOB|, consider checking for other artifacts such as ``QEMU_EFI.fd`` and ``QEMU_VARS.fd``.
+    Use these to boot the image with the ``-drive`` flag.
```
Comment (mike-scott) [inline source/user-guide/qemu/arm64.rst:27]: Did we test the `qemu-system-aarch64` block underneath here?

File: source/user-guide/qemu/arm.rst (hunk 1 above)
Comment (kprosise) [reply source/user-guide/qemu/arm.rst:12]: I will move the admonition about `-no-acpi` before that block. Regarding the substitution not working, this is a known issue, and is a limitation of rst. I believe I mentioned replacing it with the "name of your machine" when I originally wrote this, though I should make it more explicit.

File: source/user-guide/qemu/arm64.rst (hunk 2 above)
Comment (kprosise) [reply source/user-guide/qemu/arm64.rst:27]: Unsure, but I believe so, as @camilamacedo86 added it while working through this section.

Comment (camilamacedo86) [reply source/user-guide/qemu/arm64.rst:28]: I am not sure if I got your question @mike-scott 
But in the past I could make the image run within: https://foundriesio.slack.com/archives/GP2B2BKFG/p1695207237532609

Comment (mike-scott) [reply source/user-guide/qemu/arm64.rst:28]: This block looks quite different from the latest qemu block in the main section

File: source/user-guide/qemu/arm.rst [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 
-arm
-===
+``arm``
+=======
 
 .. include:: arm-substitutions.inc
+
+.. important::
+    If using QEMU version 9.0 or later, replace ``-no-acpi`` with ``-machine acpi=off``
```
Comment (camilamacedo86) [inline source/user-guide/qemu/arm.rst:9]: Should not better we keep the commands using the latest and add a note with the opposite info?

For versions < 9.0 replace `-machine acpi=off` with `-no-acpi`

File: source/user-guide/qemu/qemu-instructions.template [hunk 4]
```diff
@@ -106,27 +106,24 @@ In general, adding the following flags to the QEMU CLI should work:
 
          |QEMU_GUI_FLAGS_MAC|
 
+Also, do not copy the ``-nographic`` flag at the end of the QEMU CLI below.
```
Comment (camilamacedo86) [inline source/user-guide/qemu/qemu-instructions.template:109]: +1

**Comment by doanac** [comment]
Docs for 825d722f87fa9def1d5e14db51678514a1b830f4 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2665/docs/artifacts/html/index.html

**Pull request merged**